code/index.html

```diff
@@ -4,19 +4,169 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header">
+
+           <div class="navigationbar"> <nav>
+          <a href="#home" class="active"><picture>
+            <img src="./logo.svg" alt="technigo">
+           </picture></a>
+          <div id="myLinks">
+           <a href="#">SIGTHSEEING</a>
+           <a href="#">RESTAURANTS</a>        
+           <a href="#">BARS AND CLUBS</a>
+           <a href="#">CULTURE</a>
+           <a href="#">EVENTS</a>
+          </div>
+          <a href="javascript:void(0);" class="icon" onclick="myFunction( )">
+            <i class="fa fa-bars"></i>
+          </a>
+          </nav>
+          </div>
+
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+
+    <section class="large-news">
+      <div><picture>
```

Comment:
Might just be a preference thing, but I would bring the opening picture tag to line 35. It makes the code much easier to read (in my opinion)